src/registry_schemas/flask/schema_services.py

```diff
@@ -67,7 +67,7 @@ def rsbc_filing_schema_store(self) -> dict:
         :return: dict
         """
         if 'rsbc_filing_schema_store' not in g:
-            g.rsbc_filing_schema_store = get_schema_store()
+            g.rsbc_filing_schema_store = get_schema_store()  # pylint: disable=assigning-non-slot
```

Comment:
As we are unable to modify flask's g class to include a slot for `g.rsbc_filing_schema_store`, have disabled pylint warning for `assigning-non-slot`